archetypes/Pools/FilterSelects/Bar.tsx

```diff
@@ -41,143 +43,175 @@ const REBALANCE_OPTIONS_MOBILE = [
+                                ticker: (key !== 'All' ? key : '') as LogoTicker,
+                            }))}
+                            onSelect={(val) => dispatch({ type: 'setLeverageFilter', leverage: val as LeverageEnum })}
+                        />
+                        <SearchInput
+                            className="w-60"
+                            placeholder="Search"
+                            value={state.search}
+                            onChange={(search) => dispatch({ type: 'setSearch', search })}
+                        />
+                    </div>
+                    <TWButtonGroup
+                        value={state.deltaDenotation}
+                        onClick={(option) => dispatch({ type: 'setDenotation', denotation: option as DeltaEnum })}
+                        color="greyed"
+                        border="rounded"
+                        borderColor="greyed"
+                        options={DENOTATION_OPTIONS}
                     />
-                </div>
-                <HiddenExpand className="lg:hidden" defaultHeight={0} open={state.filtersOpen}>
+                </HiddenExpand>
+
+                <div className="mt-5 flex flex-col md:flex-row md:justify-between md:items-center">
                     <div className="flex">
-                        <div className="flex mr-4 flex-col">
-                            <span className="inline-flex">
-                                <ArrowDown className="rotate-180 text-green-600 h-6" />
-                                <ArrowDown className="text-red-600 mr-1 h-6" />
-                                <h3 className="mb-1 text-theme-text">Denotion</h3>
-                            </span>
-                            <div className="mt-auto ml-auto">
-                                <TWButtonGroup
-                                    value={state.deltaDenotion}
-                                    onClick={(option) =>
-                                        dispatch({ type: 'setDenotion', denotion: option as DeltaEnum })
-                                    }
-                                    color="greyed"
-                                    border="rounded"
-                                    borderColor="greyed"
-                                    options={DENOTION_OPTIONS}
-                                />
-                            </div>
+                        <div className="xl:hidden">
+                            <TWButtonGroup
+                                value={state.rebalanceFocus}
+                                onClick={(option) =>
+                                    dispatch({ type: 'setRebalanceFocus', focus: option as RebalanceEnum })
+                                }
+                                color="tracer"
+                                options={REBALANCE_OPTIONS_MOBILE}
+                            />
+                        </div>
+                        <div className="hidden xl:block">
```

Comment:
xl is quite a large breakpoint, from the designs I only see mobile and tablet, maybe ask Izzy if these break points are fine. Might need to reduce it to md otherwise it will swap to mobile view far to early

Reply:
will need to wrap or hide some items to make it lg or md, or we can remove the historic data tab since it says coming soon anyway